### PR TITLE
Add cert path for collectors for proxied SSL verification

### PR DESCRIPTION
**Pull request by chrisuehlinger-techslice**

### CODE REVIEW

File: scrapeomatic/collector.py [hunk 1]
```diff
@@ -52,11 +53,12 @@ def make_request(self, url, params=None, headers=None):
             headers = {}
         if params is None:
             params = {}
+
+        proxy_dict = None
         if self.proxy:
             proxy_dict = {
                 'http': f'http://{self.proxy}',
                 'https': f'http://{self.proxy}'
```
Comment (cgivre) [inline scrapeomatic/collector.py:61]: Should this be `https` here as well?

**Review by cgivre — COMMENTED** [review]
One minor question but other than that LGTM.

File: scrapeomatic/collector.py (hunk 1 above)
Comment (cgivre) [reply scrapeomatic/collector.py:61]: @chrisuehlinger-techslice Could you please rebase on current master as well?

Comment (chrisuehlinger-techslice) [reply scrapeomatic/collector.py:61]: No, at least according to the docs I've been looking at. And having them both be http worked for accessing Instagram which serves over https

**Pull request merged**